build/pause/Makefile

```diff
@@ -12,11 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-.PHONY: all push push-legacy container clean orphan
+.PHONY: all push container clean orphan all-push push-manifest
+
+include ../../Makefile.manifest
```

Comment:
this could also be just `../root/Makefile.manifest`, why not that?

Reply:
make sense, anyways I don't have to expose this makefile in the root directory